.github/workflows/ccpp.yml

```diff
@@ -168,7 +168,7 @@ jobs:
 
   build-and-check-gcc-48:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
That un-does your earlier commit 19d5f921d0ce536e0e9419b3cfc5e5c2fcef9e33 "ci: gcc-4.8 package is only available in Ubuntu 18.04".  Is that intentional?

Reply:
The diff looks backward. I did try to lock the gcc4.8 job to 20.04 as well, but the package is not available there. This caused the CI for [f7172bc](https://github.com/Rust-GCC/gccrs/pull/1688/commits/f7172bc2892c552484d1ee67fd5eb9f5414679c5) to fail https://github.com/Rust-GCC/gccrs/actions/runs/3637494717/jobs/6138569872

so I reverted it to 18.04. I think it can be squashed/fixed-up in the earlier commit as it doesn't make a lot of sense on its own :) I'll do that once the CI passes